### PR TITLE
Add FlowKit, FlowUtils and FlowIO feed-stocks

### PR DESCRIPTION
**Pull request by gergelycsegzi**
Recipes built using greyskull online for the pip packages of the same name

@conda-forge-admin, please ping conda-forge/help-python

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [?] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [-] If static libraries are linked in, the license of the static library is packaged.
- [?] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gergelycsegzi** [comment]
From the failed logs it's unclear to me why `ERROR: Could not find a version that satisfies the requirement numpy>=1.19 (from versions: none)` ([here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=448756&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=4378)) would apply when it shows a correct [higher version cached before](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=448756&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=4224).

Thanks for helping in advance!

**Comment by gergelycsegzi** [comment]
@conda-forge-admin, please ping conda-forge/help-python

any help would be appreciated in resolving [the issue](https://github.com/conda-forge/staged-recipes/pull/17759#issuecomment-1023668138)

**Comment by gergelycsegzi** [comment]
Trying without flowkit, with just the two base libraries

**Comment by gergelycsegzi** [comment]
Seems like FlowKit recipe was the issue so I will open a separate PR for that package. It relies on these 2 packages in it's requirements so let me know if I need to wait for this PR to merge before that could work.

File: recipes/flowutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy >=1.17
```
Comment (xhochy) [inline recipes/flowutils/meta.yaml:24]: You already have a `pin_compatible` below that will pin at least that high.
```suggestion
```

Comment (gergelycsegzi) [reply recipes/flowutils/meta.yaml]: Thanks, fixed. Any pointers on this?

> Seems like FlowKit recipe was the issue so I will open a separate PR for that package. It relies on these 2 packages in it's requirements so let me know if I need to wait for this PR to merge before that could work.

**Comment by gergelycsegzi** [comment]
Gentle ping
@conda-forge-admin, please ping conda-forge/help-python

**Pull request closed without merging**